core/src/main/scala/higherkindness/mu/rpc/srcgen/compendium/CompendiumClient.scala

```diff
@@ -0,0 +1,55 @@
+package higherkindness.mu.rpc.srcgen.compendium
+
+import cats.effect.Sync
+import org.http4s._
+import org.http4s.circe._
+import cats.implicits._
+import org.http4s.client.Client
+
+trait CompendiumClient[F[_]] {
+
+  /** Retrieve a Protocol by its id
+   *
+   * @param identifier the protocol identifier
+   * @param version    optional protocol version number
+   * @return a protocol
+   */
+  def retrieveProtocol(identifier: String, version: Option[Int]): F[Option[RawProtocol]]
```

Comment:
Should we make the version a String or something, in anticipation of https://github.com/higherkindness/compendium/pull/284 ?